src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -85,38 +91,38 @@
     private static final double BLOCK_BREAK_SOUND_RANGE = 16.0D;
 
     /**
-     * Modifier to lower the sound of block breaks.
-     * <p>
-     * Decrease this to make sounds louder.
+     * The range in which someone will see the particles from a block breaking.
      */
-    private static final double BLOCK_BREAK_SOUND_DAMPER = 8.0D;
+    private static final double BLOCK_BREAK_PARTICLE_RANGE = 16.0D;
 
     /**
-     * The height of half a block.
+     * Divide experience by a factor to ensure more levels fit in an int.
      */
-    private static final double HALF_BLOCK = 0.5D;
+    private static final int EXP_DIVIDER = 10;
 
     /**
-     * Modifier for sound frequency when breaking blocks.
+     * Chance the citizen will rant about bad weather. 20 ticks per 60 seconds = 5 minutes.
      */
-    private static final double SOUND_FREQ_MODIFIER = 0.5D;
-
+    private static final int    RANT_ABOUT_WEATHER_CHANCE = 20 * 60 * 5;
     /**
-     * The range in which someone will see the particles from a block breaking.
+     * Height of the citizen.
      */
-    private static final double BLOCK_BREAK_PARTICLE_RANGE = 16.0D;
-
+    private static final double CITIZEN_HEIGHT            = 1.8D;
     /**
-     * Divide experience by a factor to ensure more levels fit in an int.
+     * Width of the citizen.
      */
-    private static final int EXP_DIVIDER = 10;
-    private static Field        navigatorField;
-    protected Status status = Status.IDLE;
-    private RenderBipedCitizen.Model modelId = RenderBipedCitizen.Model.SETTLER;
+    private static final double CITIZEN_WIDTH             = 0.6D;
+    /**
+     * Defines how far the citizen will be rendered.
+     */
+    private static final double RENDER_DISTANCE_WEIGHT    = 2.0D;
+    private static Field navigatorField;
+    protected Status                   status  = Status.IDLE;
+    private   RenderBipedCitizen.Model modelId = RenderBipedCitizen.Model.SETTLER;
     private String           renderMetadata;
     private ResourceLocation texture;
     private InventoryCitizen inventory;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Private field 'inventory' could be made final; it is only initialized in the declaration or constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3AImmutableField)